test/upgrade/service_postupgrade_test.go

```diff
@@ -62,6 +74,30 @@ func TestBYORevisionPostUpgrade(t *testing.T) {
 	}
 }
 
+func configUpdated(t *testing.T, clients *test.Clients, serviceName string) bool {
+	configObj, err := clients.ServingAlphaClient.Configs.Get(serviceName, metav1.GetOptions{})
```

Comment:
Can you make an issue to migrate from Alpha clients in these tests to the v1 clients?

All of our versions we are testing upgrades to/from should now support v1.

Reply:
Sounds good. Just created the issue https://github.com/knative/serving/issues/6816